soc/arm/microchip_mec/mec1501/Kconfig.soc

```diff
@@ -66,3 +66,9 @@ config SOC_MEC1501_EXT_32K_PARALLEL_CRYSTAL
 
 	  Say n if the crystal is connected single ended to
 	  the XTAL2 pin or a 32KHz square wave is on XTAL2.
+
+config SOC_MEC1501_VTR3_1_8V
+	bool "VTR3 power rail is tied to 1.8V"
+	default n
```

Comment:
`bool` symbols implicitly default to `n`, so can remove this.

Generates nicer docs without the `default n` too ("Implicitly defaults to n" instead of "n is <bunch of dependencies>").